packages/expo-notifications/plugin/src/withNotificationsAndroid.ts

```diff
@@ -154,9 +180,49 @@ async function removeNotificationIconImageFilesAsync(projectRoot: string) {
   );
 }
 
-export const withNotificationsAndroid: ConfigPlugin = config => {
-  config = withNotificationIconColor(config);
-  config = withNotificationIcons(config);
-  config = withNotificationManifest(config);
+/**
+ * Save sound files to <project-root>/android/app/src/main/res/raw
```

Comment:
```suggestion
 * Save sound files to `<project-root>/android/app/src/main/res/raw`
```

MD formatting is a bit nicer in doc block and doc gen.